### ISSUE TITLE
sleeping players mob exploitation

### ISSUE THREAD

**Issue by Captaincardgage** [ParadiseSS13/Paradise#6681]
**Problem Description**:
What is the problem?
Mobs will not effect players if they're sleeping. so if a player falls asleep, not resting. the mob will not harm them.

**What did you expect to happen**: 
Why do you think this is a bug?
You'd expect the mob to still harm players despite asleep or not.

**What happened instead**:
How is what happened different from what you expected?
The mob had ignored the sleeping player instead had decided to attack

**Why is this bad/What are the consequences:**
Why do you think this is an important issue?
1. Exploitation: People can use this to not get harmed by the mobs if a xenobio let them on a rampage or someone threw manhack or feral cat delivery grenades. Allowing players to not having to fight any of them.

**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.
1. Spawn any form of hostile mob. 
(slime excluded since they have a special to just eat the bodies sleeping or not)
2. While running, goto the IC tab and select sleep. Where the mob will ignore the body and go find a awake/resting mob/player

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Anything else you can tell us.
Testing via private server.

**Comment by KasparoVy** [comment]
Feature :). Pretending to be dead.

**Comment by Captaincardgage** [comment]
I think a few of these mobs know when a persons dead or not. especially during mid chase. But I can see it as a feature but when you see a load of terrorspiders or really anything killing people running down the hall you'd expect them to make sure whatever is on the floor is truly dead

**Comment by jughu** [comment]
Why would a manhacker kill something whats already gasping and litteraly dying when they can do it to others too

**Comment by FalseIncarnate** [comment]
Fairly sure this is intended so that mobs will ignore SSD characters (may be wrong on that intent), but I wouldn't say this is much of an exploit since it takes a bit to wake up and the mob might just hang around... waiting.

**Comment by Lady-Luck** [comment]
Finally narcolepsy pays off.

**Comment by Captaincardgage** [comment]
I can see the two tides here, It can be used for just exploit so you don't get killed by that blobbernaut who just smashed the wall in. But then again it does preserve the SSD's and in a rp scenario could work well.

**Comment by taukausanake** [comment]
Maybe a delay time for the sleep verb could be nice. It's weird when you say "Ima fall asleep now!" and then conk you're sleeping. Like it takes time to really fall asleep and when you do then it's good. Getting poked or bumped while trying to do this should cancel it because annoying when it happens in real life

**Comment by Markolie** [comment]
This is indeed a feature request unfortunately enough.

**Comment by KasparoVy** [comment]
Nullifies playing dead as a usable tactic. Normally if you have to resort to doing this you've got a really good chance to get fucked up anyway.

**Comment by Anasari** [comment]
I personally like the idea of being able to sleep to avoid hostile mobs. IT's saved my life man time from encounter I cannot flee from

**Comment by davidchan13** [comment]
Feature and not one that's as easy to abuse as OP would like you to believe.

Mobs loiter in their current area more so than old singularity. Unless you have someone else to draw the mob away youd need to log out for a good 10 minutes or more to be reasonably safe.

Sleep renders you blind and immobile for a time, but there is a 5 second window or more where you count as awake but can not move or see before standing up. Mobs can and will attack you in this window and some mobs can crit you in this time and keep you down.

You also are still under the effects of environmental hazards so if carp broke in the windows and you sleep, the pressure is going to get you along with the suffocation. Possibly can be seen as an unmarked buff for Vox, Plasmemes and Diona but not one worth getting upset about.

Fast acting players could maybe rage quit to avoid death and come back later but for most it's usually better to run or fight if they got the means to do so.

TL;DR this only serves to protect SSD characters. There is no reliable way to use this to survive mob attacks without meta comms

**Comment by sothangel** [comment]
I have seen people hit the reconnect button to avoid Beepsky/ED arrests before, which can't really be changed much without screwing over legit SSD people.
